ca/config.go

```diff
@@ -157,6 +157,11 @@ func (s *SecurityConfig) UpdateRootCA(rootCA *RootCA, externalCARootPool *x509.C
 
 	s.rootCA = rootCA
 	s.externalCAClientRootPool = externalCARootPool
+
+	s.externalCA.mu.Lock()
+	s.externalCA.rootCA = rootCA
+	s.externalCA.mu.Unlock()
```

Comment:
Not seeing the setter used here, but GitHub seems to be acting up, so maybe it's not showing the full change.